m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -2292,6 +2398,14 @@ PROCEDURE READCONFIGFILESETTINGS
            PARAMALLOWABLESHARECODES(SHARECODECLUB)=TRUE
           END
         END
+[* Share ID optional on new member transfers 
+*]
+       ELSE IF PARAMNAME="SIDO" AND
```

Comment:
Simplification ??

```
       ELSE IF PARAMNAME="SIDO" AND
               CHARACTERSEARCH("1YESTRUE",UPPERCASE(PARAMVALUE))>0  THEN

          PARAMSIDOPTIONAL=TRUE

```

Reply:
As in CHARACTERSEARCH will never return a value greater than zero if PARAMVALUE=""??  So need to check for PARAMVALUE<>""??  Makes sense.  I suppose I should fix the other ones that do a similar thing.  I'll fix 'em.

Reply:
Made updates.